src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -849,6 +849,10 @@ public enum InputSelectionMode {
   @Option("If true, use JUnit's reflective invocation; if false, use direct method calls")
   public static boolean junit_reflection_allowed = true;
 
+  /** Whether Randoop's generated tests will be run under JUnit 4.11 or earlier. */
+  @Option("True if generated JUnit tests will be run under JUnit 4.11 or earlier.")
+  public static boolean junit_pre_4_12 = false;
```

Comment:
I suggest removing this command-line option.  I see no harm in always including the definition, which should work for all versions of JUnit.